dissect/target/plugins/os/windows/tasks.py

```diff
@@ -175,6 +182,169 @@
+        uint16      last_year;                          /* 0x34 - last run year. */
+        uint16      last_month;                         /* 0x36 - last run month. */
+        uint16      last_weekday;                       /* 0x38 - last run weekday. */
+        uint16      last_day;                           /* 0x3a - last run day of the month. */
+        uint16      last_hour;                          /* 0x3c - last run hour (24h). */
+        uint16      last_minute;                        /* 0x3e - last run minute. */
+        uint16      last_second;                        /* 0x40 - last run second. */
+        uint16      last_millisecond;                   /* 0x42 - last run millisecond. */
+        uint16      running_instances;                  /* 0x44 - number of currently running instances. */
+        uint16      app_name_len;                       /* 0x46 - app name character count. */
+        char        app_name[app_name_len * 2];         /* 0x48 - app name - null-terminated Unicode string. */
+        uint16      par_char_count;                     /*      - parameters character count. */
+        char        parameters[par_char_count * 2];     /*      - parameters - null-terminated Unicode string. */
+        uint16      dir_char_count;                     /*      - working dir character count. */
+        char        working_dir[dir_char_count * 2];    /*      - working dir - null-terminated Unicode string. */
+        uint16      author_char_count;                  /*      - author character count. */
+        char        author[author_char_count * 2];      /*      - author - null-terminated Unicode string. */
+        uint16      comment_char_count;                 /*      - comment character count. */
+        char        comment[comment_char_count * 2];    /*      - comment - null-terminated Unicode string. */
+        uint16      user_data_size;                     /*      - user data size in bytes. */
+        uint8       user_data[user_data_size];          /*      - arbitrary bits, implementation specific. */
+        uint16      reserved_data_size;                 /*      - should be 0 or 8. */
+        HRESULT     reserved_hresult;                   /*      - used to describe an error. */
+        uint32      reserved_task_flags;                /*      - not used, should be zero. */
+        uint16      trigger_count;                      /*      - size in bytes of array of triggers. */
+        TRIGGER     task_triggers[trigger_count];       /*      - an arry of zero or more triggers. */
+//      uint16      s_ver;                              /*      - SignatureVersion, should be 1. */
+//      uint16      c_ver;                              /*      - MinClientVersion, should be 1. */
+//      uint8       job_signature[64 * s_ver * c_ver];  /*      - calculated job signature. */
+    };
+    """
+atjob = cstruct.cstruct()
+atjob.load(c_atjob)
```

Comment:
```suggestion
c_atjob = cstruct.cstruct()
c_atjob.load(atjob_def)
```
To be more consistent with other code.